setup_venv.sh

```diff
@@ -64,6 +65,7 @@ elif [[ $(uname -s) = 'Linux' ]]; then
   echo "${Yellow}Linux detected"
   if [ "$PYTHON_VERSION_X_Y" == "3.7" ] || [ "$PYTHON_VERSION_X_Y" == "3.8" ]  || [ "$PYTHON_VERSION_X_Y" == "3.9" ] || [ "$PYTHON_VERSION_X_Y" == "3.10" ] ; then
     torch_mlir_bin=true
+    install_tensorflow=true
```

Comment:
I think requirements-importer.txt should already bring in tf no ?

Reply:
yeap, you're right

Reply:
done!